search-extension/src/main/java/com/afrigis/services/search/extension/SuburbRiskTotalResponse.java

```diff
@@ -11,5 +11,34 @@
  */
 public interface SuburbRiskTotalResponse extends Response {
 
-    public double getTotalRisk();
+    /**
+     * Get totalRisk from AfriGIS service call
+     *
+     * @return double
+     * @throws java.lang.Exception in case the service call was invalid
+     * according to the internal AfriGIS service API, cannot return 0 as this
+     * would mean the service call was a success
+     */
+    public double getTotalRisk() throws Exception;
+
+    /**
+     * AfriGIS service call response, indicates status of service call
+     *
+     * @return code
+     */
+    public int getCode();
```

Comment:
@maposatj correct me if I'm wrong, but don't we inherit these methods from `Response` ?